packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1 +1,15 @@
-//Styles for off-canvas editor, remove this line when you add some css to this file!
+.offcanvas-editor-list-view-tree-wrapper {
+	max-width: 100%;
+	overflow-x: auto;
+	.block-editor-list-view-block__menu-cell:last-child {
+		padding-right: 0;
+	}
+
+	.block-editor-list-view-leaf {
+		display: block;
+		max-width: $sidebar-width - (2 * $grid-unit-20);
+		.block-editor-block-icon {
+			margin-right: 4px;
```

Comment:
This is an arbitary number. Can we explain in a comment why it's needed and what it represents? 

I appreciate that seems very nit-picky but it helps avoid people randomly amending values in future without knowing what they might affect 🙇 🙇 🙇 🙇 🙇

Reply:
I think with the change to the max-width, tighting those spaces this way may not be needed and we could leave it as it was.